config/routes.rb

```diff
@@ -30,6 +30,13 @@
       end
     end
 
+    resource :refund, only: [:create, :new], path: "/refund" do
+      RefundPagesManager.page_names.each do |page|
+        resource page.underscore, only: %i<show update>, controller: :refunds,
```

Comment:
I think we agreed not to use %i :)

Reply:
Well spotted - copied from original before the change I bet